### PR TITLE
Create from scratch process 'shorten' + removing create scratch view

### LINKED ISSUE #2926: Improve create empty map
Hi guys, one thing. @carlostallon @saleiva @xavijam 
image

In create empty map, what do you think about change the link for a button and go directly to a empty map. Removing this view ( i thinks it's no necessary)

image

### PR DESCRIPTION
**Pull request by xavijam**
- Create empty map/dataset/layer doesn't need two clicks, only one.
- Button should appear disabled if user can't create more datasets.

---
Adding a new layer:
screen shot 2015-04-28 at 11 12 04

---
Creating a new map:
screen shot 2015-04-28 at 11 12 33

---
Creating a new dataset:
screen shot 2015-04-28 at 11 12 15

Fixes #2926 

cc @piensaenpixel 

Could you take a look @viddo?

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3319/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3320/))

File: app/assets/stylesheets/new_common/create/listing/listing_navigation.css.scss [hunk 1]
```diff
@@ -27,3 +27,12 @@
...
 .Filters.Filters--navListing .Filters-searchInput { margin-top: -2px } /* hacking line-height between search and input */
+.Filters--navListing .CreateButton {
+  height: 34px;
+  margin-top: 48px;
+  font-size: $sFontSize-small;
+}
+.Filters--navListing .CreateButton.is-disabled:hover {
+  background: none;
+  color: $cTypography-link;
+}
```
Comment (viddo) [inline app/assets/stylesheets/new_common/create/listing/listing_navigation.css.scss]: Shouldn't we try to keep styles isolated per component than cross referencing them like this? I.e. define a suitable `--modifier` for this kind of button, e.g. `.Button--create` or such?

**Comment by viddo** [comment]
:ok_hand:

File: app/assets/stylesheets/new_common/create/listing/listing_navigation.css.scss (hunk 1 above)
Comment (xavijam) [inline app/assets/stylesheets/new_common/create/listing/listing_navigation.css.scss]: Agree, thanks!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3321/))

**Pull request merged**